src/daemon/rpmostreed-transaction-livefs.c

```diff
@@ -522,6 +522,106 @@ replace_subpath (OstreeRepo *repo,
   return TRUE;
 }
 
+/* The sledgehammer 🔨 approach. Because /usr is a mount point, we can't replace
+ * all of it. We could do a diff, but doing that precisely and quickly depends
+ * on https://github.com/ostreedev/ostree/issues/1224
+ *
+ * In this approach, we iterate over and exchange subdirectories. Some issues
```

Comment:
If we're not recursing, let's clarify that it's only first-level subdirectories of `/usr`.